### ISSUE TITLE
Hunter no release hook

### ISSUE THREAD

**Issue by duralakun** [necropotame/teeworlds-infclass#160]
When a hunter hooks a human, the hunter will not be able to release the hook until the hook time has run out.

**Comment by necropotame** [comment]
This is a bug or a feature?

**Comment by duralakun** [comment]
A feature :P

**Comment by necropotame** [comment]
I think it's better for the undead. Even dead, he stay stick to you

**Comment by duralakun** [comment]
i like it

**Comment by necropotame** [comment]
I don't think that force him is a good idea. We can keep it as it is when he is alive, but once dead, he can keep his pray for one or two seconds.

**Comment by duralakun** [comment]
ok, can we force the hunter ?

**Comment by necropotame** [comment]
It will just make him weaker
